### PR TITLE
Fixes #8166: Add filters on compliance tables to hide some kind of compliance

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://issues.rudder.io/issues/8166

Addition of a new filter based on compliance status. Users can now decide which statuses to hide or show.
 
compliance-filter-1
compliance-filter-2

### CODE REVIEW

**Comment by RaphaelGauthier** [comment]
PR rebased

File: webapp/sources/rudder/rudder-web/src/main/elm/sources/Rules/ViewUtils.elm [hunk 1]
```diff
@@ -141,52 +160,60 @@ valueCompliance =
...
+            |> List.fvalueComplianceilter (filterByCompliance complianceFilters)
+            |> List.sortWith sortFunction
+            |> List.map Left
         Value c ->
           let
             sortFunction =  subItemOrder subFun model sortId
           in
-             List.map Right (List.sortWith sortFunction c.values)
+            c.values
+            -- @TODO : |> List.filter someFunction
```
Comment (RaphaelGauthier) [inline webapp/sources/rudder/rudder-web/src/main/elm/sources/Rules/ViewUtils.elm:214]: The last thing missing I think, is the filter at this level. I haven't managed to write anything functional...

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**